packages/components/src/item-group/styles.ts

```diff
@@ -10,7 +10,7 @@ import { CONFIG, COLORS } from '../utils';
 
 export const unstyledButton = css`
 	appearance: none;
-	border: 1px solid transparent;
+	border: 0;
```

Comment:
I wonder why this was using a transparent border

Reply:
I wonder the same. It could be related to vertical height/metrics. In any case removing it allowed the border to go edge to edge instead of stopping in a triangular joint.

Reply:
This change currently breaks focus styles (see the `&:focus` rule a few lines below).

Leaving a transparent border makes allows us to apply a `border-color` change for focus styles, without having the component change its outer dimensions.